scripts/wildcard-domain-processor/index.ts

```diff
@@ -0,0 +1,17 @@
+import { wildcardDomainProcessor } from './wildcard-domain-processor';
+import { patchPlatforms } from './platforms-patcher';
+
+// FIXME check if this variable already used somewhere
+const FILTERS_DIR = '../../filters';
+const WILDCARD_DOMAINS_JSON = 'wildcard_domains.json';
+const PLATFORMS_DIR = '../../platforms';
+
+(async (): Promise<void> => {
+    try {
+        // await wildcardDomainProcessor(FILTERS_DIR, WILDCARD_DOMAINS_JSON);
```

Comment:
I think we'll need to make domains extraction available for manual run for now (in the future may automate this so TODO would be appropriate).